### ISSUE TITLE
Can the BSC node be started through a cron job?

### ISSUE THREAD

**Issue by daica75** [binance-chain/bsc#322]
Can the BSC node be started through a cron job?  I have tried running the geth command through cron and running it as a background process by added an & at the end, and whenever I do that, it doesn't seem like the BSC node started up correctly, and I do not see the geth.ipc file.  But when I just run it as I usually do, I can see the geth.ipc file.  Is there some other option I need to add to run geth so that it would start correctly?

I am currently running the following to start the node:
geth --config ./config.toml --datadir ./node  --cache 18000  --rpc.allow-unprotected-txs --txlookuplimit 0 console

**Comment by j75689** [comment]
Have you seen any Error log messages?
The log can be exported by configuring in config.toml.
```
[Node.LogConfig]
FileRoot = "/server/logs/"
FilePath = "bsc.log"
MaxBytesSize = 10485760
Level="information"
```

**Comment by daica75** [comment]
Sorry, my node crashed and I am waiting for it to sync up again :(  I do have the logging set, and I don't remember seeing any errors.  But once it is up again I will be able to check.  Basically what I was finding is that if I have the bsc node started through cron or running with & to run as a background process, it didn't seem like the bsc node was starting correctly.

**Comment by Crypto2** [comment]
You shouldn't need 'console' at the end for a cron job and depending on your PATH variable you may need to use ./geth instead of just geth.

**Comment by koen84** [comment]
You don't need `console` at the end.
You can run it in tmux / screen.
You can run it via systemd unitfile.

**Comment by daica75** [comment]
Hi.  You are right.  I found that I ran it with console for start in cron.d and it did not work.  I ran the script without console and it worked, and I was able to run geth attach afterwards to it to see the console.  Also, I found instructions to run the script to start geth as a service.  I am all set now.  I had some power issues afterwards that caused my node to be out of sync again and that was why I took so long to respond.  Thank you again!  Also, I had to reference the absolute path to geth since it was not in the path when running the script through cron or as a service, and that may have been part of the issue as well.
